File: golang/go-demo/basic/if.go
package main

import "fmt"

func main() {
	condition := true

	// the parenthese following if keyword can be omitted
	if condition {
		fmt.Println("yes!")
	} else {
		fmt.Println("no!")
	}

	// else if
	integer := 1
	if integer == 2 {
		fmt.Println("It's two.")
	} else if integer == 1 {
		fmt.Println("It's one.")
	} else if integer != 1 && integer != 2 {
		fmt.Println("It's other number.")
	}
}

File: rust/rust-sample/examples/if.rs
fn main() {
    // if else
    let i = 2;
    if i <= 10 {
        println!("{i} <= 5");
    } else {
        println!("{i} > 5");
    }

    // if else if
    if i == 1 {
        println!("one");
    } else if i == 2 {
        println!("two");
    } else if i == 3 {
        println!("three");
    }

    let if_i_eq_3 = if i == 3 { "true" } else { "false" };
    println!("{if_i_eq_3}");
}
